syntax = "proto3";
import "common.proto";
package normal;
option cc_generic_services = true;

message SceneTestRequest
{
}

message SceneTestResponse
{
	ClientError error = 1;
}


service PlayerSceneService
{
  rpc EnterScene (SceneTestRequest) returns (SceneTestResponse);
}